CouchBaseStorage/CouchBaseStorageProvider.cs

```diff
@@ -61,12 +64,17 @@ public class CouchBaseDataManager : IJSONStateDataManager
         /// </summary>
         protected IBucket bucket;
 
+        /// <summary>
+        /// 

```

Comment:
Empty summary. The code documentation is not complete so I'd be happy to either add something or remove the summary.